src/pages/projects.js

```diff
@@ -357,6 +360,10 @@ const ProjectDescriptions = () => (
             <a href="https://github.com/iovisor/gobpf">gobpf</a> is a CGo-based
             library which provides Go bindings for the BCC framework as well as
             low-level routines to load and use eBPF programs from ELF files.
+            <a href="https://github.com/aquasecurity/libbpfgo">libbpfgo</a> is a
+            Go wrapper around libbpf. It supports BPF CO-RE and its goal is to 
+            be a complete implementation of libbpf APIs. It uses CGO to call 
```

Comment:
```suggestion
            be a complete implementation of libbpf APIs. It uses CGo to call 
```
capitalization consistency with line 360

Reply:
Done